imports/plugins/core/accounts/client/templates/wallet/wallet.js

```diff
@@ -0,0 +1,123 @@
+import {
+  Meteor
+} from "meteor/meteor";
+import {
+  Template
+} from "meteor/templating";
+import {
+  Accounts,
+  Wallets
+} from "/lib/collections";
+import verifyPayment from "./verifyPayment";
+
+let paystackKeys = {};
+
+Template.wallet.onCreated(function bodyOnCreated() {
+  /* eslint-disable no-undef */
+  this.state = new ReactiveDict();
+  this.state.setDefault({
+    details: {
+      balance: 0
+    }
+  });
+  this.autorun(() => {
+    this.subscribe("myTransactions", Meteor.userId());
+    const transactionInfo = Wallets.find().fetch();
+    if (transactionInfo.length > 0) {
+      this.state.set("details", transactionInfo[0]);
+    }
+  });
+});
+
+const finalizeDeposit = (paystackMethod) => {
+  Meteor.call(
+    "wallet/transaction",
+    Meteor.userId(),
+    paystackMethod.transactions,
+    (err, res) => {
+      if (res) {
+        Alerts.toast("Your deposit was successful", "success");
+      } else {
+        Alerts.toast("A error occured, please try again", "error");
+      }
+    }
+  );
+};
+
+/**
+ * @returns {funct} handleDepositPayment
+ * @param {void} result
+ */
+
+Template.wallet.events({
+
+  "submit #deposit": (event) => {
+    event.preventDefault();
+    const accountDetails = Accounts.find(Meteor.userId()).fetch();
+    const userMail = accountDetails[0].emails[0].address;
+    const amount = parseInt(document.getElementById("depositAmount").value, 10);
+    if (amount < 0) {
```

Comment:
This `if` block can be merged with the one on line 63 to read
```js
if(!amount || amount <= 0) {
    Alerts.toast("Please enter a valid deposit amount", "error");
}
```

Where `!amount` is the same as writing `amount === ""`  and `amount === 0`